bin/package/installation/default

```diff
@@ -2,6 +2,7 @@
 # Define additional args for mysterium_server (see help for full list)
 CONF_DIR="--config-dir=/etc/mysterium-node"
 RUN_DIR="--runtime-dir=/var/run/mysterium-node"
-DISCOVERY=""
+DAEMON_OPTS="--tequilapi.address=0.0.0.0"
+SERVICE_OPTS="openvpn"
```

Comment:
We are not going to support wireguard?

Reply:
I wanna make just 'openvpn' as default to Raspberry package